.github/workflows/auto-approve-dependabot.yml

```diff
@@ -0,0 +1,27 @@
+# Automatically approve PRs made by Dependabot
+#
+# Written to look at the original author of the PR (instead of the current
+# actor) in order to be able to backresolve existing PRs using this action (by
+# mass labeling them). Leads to slightly unnecessary spammage of aprovals in a
+# PR...
+#
+# Only does approvals! A different GitHub Action takes care of merging.
+name: Auto-approve Dependabot
+on:
+  pull_request:
+    types:
+      - labeled
+      - opened
+      - ready_for_review
+      - reopened
+      - synchronize
+      - unlabeled
+      - unlocked
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hmarr/auto-approve-action@7782c7e2bdf62b4d79bdcded8332808fd2f179cd
+      if: github.event.pull_request.user.login == 'dependabot[bot]' || github.event.pull_request.user.login == 'dependabot-preview[bot]'
+      with:
+        github-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
This will *NOT* trigger dependabot's "merge on approval" behavior, because the default `secrets.GITHUB_TOKEN` does not have sufficient permissions to trigger further automation... No web hook event will be emitted when using this token...

There are two options to solve this:
- Configure Dependabot's [`automerged_updates`](https://dependabot.com/docs/config-file/#automerged_updates), which requires an org-wide enablement of the feature (auto merging requires the repository owner to have made it possible __and__ per-repository configuration)
- Use a privileged token that has the relevant permissions to trigger further automation (aka to cause web hook events to be emitted).

Reply:
You are right that this won't trigger automatic merges. My plan was to have Mergify or a new GitHub Action merge them as soon as they are approved. 

Seems Mergify has been configured to ignore dependabot (for good reason I imagine), so we will switch to a dependabot-aware GHA soon: https://github.com/awslabs/aws-delivlib/blob/master/.github/workflows/auto-merge-prs.yml

Reply:
I tried configuring Mergify to treat Dependabot appropriately.

This is going to be the solution until we figure out what to do about GITHUB_TOKEN for GitHub Actions.